2021-11-09 08:19:58,149 | 3734 | tracing_net.net, __init__, 99 | INFO | set up on-demand mininet
2021-11-09 08:19:58,151 | 3734 | tracing_of_pipeline, start, 125 | INFO | OpenFlow Tracer Start
2021-11-09 08:19:58,156 | 3734 | tracing_net.net, start, 312 | INFO | mininet start
2021-11-09 08:19:58,157 | 3734 | tracing_of_pipeline.analyzer, start_analyzing, 143 | INFO | start analyzing interval (start_time = 1636445998.157871, interval = 2)
2021-11-09 08:19:58,776 | 3734 | tracing_net.net, start_tracing, 525 | INFO | tracing start
2021-11-09 08:19:58,819 | 3896 | tracing_net.util.packet_capture, start_capture, 65 | INFO | packet capture start on s1-eth1
2021-11-09 08:19:58,832 | 3898 | tracing_net.util.packet_capture, start_capture, 65 | INFO | packet capture start on s1-eth2
2021-11-09 08:19:58,847 | 3901 | tracing_net.util.packet_capture, start_capture, 65 | INFO | packet capture start on s2-eth2
2021-11-09 08:19:58,850 | 3734 | tracing_net.flowtable.flow_monitor, __init__, 46 | INFO | flow monitor start : s1
2021-11-09 08:19:58,862 | 3734 | tracing_net.flowtable.flow_monitor, __init__, 46 | INFO | flow monitor start : s2
2021-11-09 08:19:58,870 | 3904 | tracing_net.util.packet_capture, start_capture, 65 | INFO | packet capture start on s1-eth3
2021-11-09 08:19:59,916 | 3907 | tracing_net.flowtable.flow_monitor, read_dump_flow, 83 | WARNING | No result dump flows
2021-11-09 08:19:59,940 | 3909 | tracing_net.flowtable.flow_monitor, read_dump_flow, 83 | WARNING | No result dump flows
2021-11-09 08:20:00,160 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:00,161 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:00,161 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:00,161 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:00,163 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:00,163 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-09 08:20:00,163 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-09 08:20:00,164 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:00,931 | 3907 | tracing_net.flowtable.flow_monitor, read_dump_flow, 83 | WARNING | No result dump flows
2021-11-09 08:20:01,025 | 3909 | tracing_net.flowtable.flow_monitor, read_dump_flow, 83 | WARNING | No result dump flows
2021-11-09 08:20:01,885 | 3907 | tracing_net.flowtable.flow_monitor, read_dump_flow, 83 | WARNING | No result dump flows
2021-11-09 08:20:01,908 | 3909 | tracing_net.flowtable.flow_monitor, read_dump_flow, 83 | WARNING | No result dump flows
2021-11-09 08:20:02,159 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:02,159 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:02,160 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:02,161 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:02,163 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:02,165 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-09 08:20:02,167 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-09 08:20:02,168 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:04,158 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:04,159 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:04,160 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:04,160 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:04,163 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:04,163 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-09 08:20:04,163 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-09 08:20:04,163 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:06,166 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:06,167 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:06,168 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:06,168 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:06,170 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:06,170 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-09 08:20:06,171 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-09 08:20:06,171 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:08,169 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:08,170 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:08,170 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:08,171 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:08,173 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:08,173 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-09 08:20:08,173 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-09 08:20:08,174 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:10,159 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:10,159 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:10,160 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:10,161 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:10,162 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:10,163 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-09 08:20:10,163 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-09 08:20:10,163 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:12,159 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:12,160 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:12,161 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:12,161 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:12,163 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:12,163 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-09 08:20:12,164 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-09 08:20:12,164 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:14,165 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:14,166 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:14,167 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:14,168 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:14,170 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:14,171 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-09 08:20:14,172 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-09 08:20:14,172 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:16,173 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:16,174 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:16,175 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:16,176 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:16,177 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:16,178 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-09 08:20:16,178 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-09 08:20:16,179 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:18,166 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:18,167 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:18,167 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:18,183 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:20:18,199 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:18,200 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-09 08:20:18,201 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-09 08:20:18,201 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:20,182 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [<Msg captured_interface=s1-eth1, sniff_timestamp=1636446019.4849055, pkt=<ARP_RAW Packet>, in_port=s1-eth1, in_phy_port=s1-eth1>, <Msg captured_interface=s1-eth1, sniff_timestamp=1636446019.5880246, pkt=<ARP_RAW Packet>, in_port=s1-eth1, in_phy_port=s1-eth1>, <Msg captured_interface=s1-eth1, sniff_timestamp=1636446019.5880663, pkt=<ICMP_RAW Packet>, in_port=s1-eth1, in_phy_port=s1-eth1>], 's2-eth2': [<Msg captured_interface=s2-eth2, sniff_timestamp=1636446019.5349715, pkt=<ARP_RAW Packet>, in_port=s2-eth2, in_phy_port=s2-eth2>, <Msg captured_interface=s2-eth2, sniff_timestamp=1636446019.5349944, pkt=<ARP_RAW Packet>, in_port=s2-eth2, in_phy_port=s2-eth2>, <Msg captured_interface=s2-eth2, sniff_timestamp=1636446019.6175637, pkt=<ICMP_RAW Packet>, in_port=s2-eth2, in_phy_port=s2-eth2>], 's1-eth2': [<Msg captured_interface=s1-eth2, sniff_timestamp=1636446019.4898, pkt=<ARP_RAW Packet>, in_port=s1-eth2, in_phy_port=s1-eth2>, <Msg captured_interface=s1-eth2, sniff_timestamp=1636446019.5545619, pkt=<ARP_RAW Packet>, in_port=s1-eth2, in_phy_port=s1-eth2>, <Msg captured_interface=s1-eth2, sniff_timestamp=1636446019.606372, pkt=<ICMP_RAW Packet>, in_port=s1-eth2, in_phy_port=s1-eth2>]})
2021-11-09 08:20:20,182 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:20,183 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-09 08:20:20,183 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-09 08:20:20,184 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:22,159 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [<Msg captured_interface=s1-eth1, sniff_timestamp=1636446019.4849055, pkt=<ARP_RAW Packet>, in_port=s1-eth1, in_phy_port=s1-eth1>, <Msg captured_interface=s1-eth1, sniff_timestamp=1636446019.5880246, pkt=<ARP_RAW Packet>, in_port=s1-eth1, in_phy_port=s1-eth1>, <Msg captured_interface=s1-eth1, sniff_timestamp=1636446019.5880663, pkt=<ICMP_RAW Packet>, in_port=s1-eth1, in_phy_port=s1-eth1>], 's2-eth2': [<Msg captured_interface=s2-eth2, sniff_timestamp=1636446019.5349715, pkt=<ARP_RAW Packet>, in_port=s2-eth2, in_phy_port=s2-eth2>, <Msg captured_interface=s2-eth2, sniff_timestamp=1636446019.5349944, pkt=<ARP_RAW Packet>, in_port=s2-eth2, in_phy_port=s2-eth2>, <Msg captured_interface=s2-eth2, sniff_timestamp=1636446019.6175637, pkt=<ICMP_RAW Packet>, in_port=s2-eth2, in_phy_port=s2-eth2>], 's1-eth2': [<Msg captured_interface=s1-eth2, sniff_timestamp=1636446019.4898, pkt=<ARP_RAW Packet>, in_port=s1-eth2, in_phy_port=s1-eth2>, <Msg captured_interface=s1-eth2, sniff_timestamp=1636446019.5545619, pkt=<ARP_RAW Packet>, in_port=s1-eth2, in_phy_port=s1-eth2>, <Msg captured_interface=s1-eth2, sniff_timestamp=1636446019.606372, pkt=<ICMP_RAW Packet>, in_port=s1-eth2, in_phy_port=s1-eth2>]})
2021-11-09 08:20:22,160 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:22,161 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l1
2021-11-09 08:20:22,162 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l3
2021-11-09 08:20:22,162 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:24,159 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's2-eth2': [], 's1-eth2': []})
2021-11-09 08:20:24,185 | 3734 | tracing_of_pipeline.analyzer, analyze, 248 | WARNING | edge l1 is not terminal edge. The analysis done before may not have been done properly.
2021-11-09 08:20:24,189 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:26,158 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [<Msg captured_interface=s1-eth1, sniff_timestamp=1636446024.8294315, pkt=<ARP_RAW Packet>, in_port=s1-eth1, in_phy_port=s1-eth1>], 's2-eth2': [<Msg captured_interface=s2-eth2, sniff_timestamp=1636446024.8289483, pkt=<ARP_RAW Packet>, in_port=s2-eth2, in_phy_port=s2-eth2>], 's1-eth2': [<Msg captured_interface=s1-eth2, sniff_timestamp=1636446024.8293247, pkt=<ARP_RAW Packet>, in_port=s1-eth2, in_phy_port=s1-eth2>]})
2021-11-09 08:20:26,163 | 3734 | tracing_of_pipeline.analyzer, analyze, 248 | WARNING | edge l1 is not terminal edge. The analysis done before may not have been done properly.
2021-11-09 08:20:26,168 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:28,164 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's2-eth2': [], 's1-eth2': []})
2021-11-09 08:20:28,175 | 3734 | tracing_of_pipeline.analyzer, analyze, 248 | WARNING | edge l1 is not terminal edge. The analysis done before may not have been done properly.
2021-11-09 08:20:28,180 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:30,159 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's2-eth2': [], 's1-eth2': []})
2021-11-09 08:20:30,164 | 3734 | tracing_of_pipeline.analyzer, analyze, 248 | WARNING | edge l1 is not terminal edge. The analysis done before may not have been done properly.
2021-11-09 08:20:30,175 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:32,164 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's2-eth2': [], 's1-eth2': []})
2021-11-09 08:20:32,165 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:32,166 | 3734 | tracing_of_pipeline.analyzer, analyze, 248 | WARNING | edge l1 is not terminal edge. The analysis done before may not have been done properly.
2021-11-09 08:20:32,169 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:34,164 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's2-eth2': [], 's1-eth2': []})
2021-11-09 08:20:34,165 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:34,166 | 3734 | tracing_of_pipeline.analyzer, analyze, 248 | WARNING | edge l1 is not terminal edge. The analysis done before may not have been done properly.
2021-11-09 08:20:34,169 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:36,159 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's2-eth2': [], 's1-eth2': []})
2021-11-09 08:20:36,161 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:36,162 | 3734 | tracing_of_pipeline.analyzer, analyze, 248 | WARNING | edge l1 is not terminal edge. The analysis done before may not have been done properly.
2021-11-09 08:20:36,165 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:38,160 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's2-eth2': [], 's1-eth2': []})
2021-11-09 08:20:38,161 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:38,162 | 3734 | tracing_of_pipeline.analyzer, analyze, 248 | WARNING | edge l1 is not terminal edge. The analysis done before may not have been done properly.
2021-11-09 08:20:38,164 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:40,168 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's2-eth2': [], 's1-eth2': []})
2021-11-09 08:20:40,169 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:40,169 | 3734 | tracing_of_pipeline.analyzer, analyze, 248 | WARNING | edge l1 is not terminal edge. The analysis done before may not have been done properly.
2021-11-09 08:20:40,172 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:42,161 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's2-eth2': [], 's1-eth2': []})
2021-11-09 08:20:42,163 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:42,163 | 3734 | tracing_of_pipeline.analyzer, analyze, 248 | WARNING | edge l1 is not terminal edge. The analysis done before may not have been done properly.
2021-11-09 08:20:42,167 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:44,170 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's2-eth2': [], 's1-eth2': []})
2021-11-09 08:20:44,178 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:44,179 | 3734 | tracing_of_pipeline.analyzer, analyze, 248 | WARNING | edge l1 is not terminal edge. The analysis done before may not have been done properly.
2021-11-09 08:20:44,200 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:46,164 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's2-eth2': [], 's1-eth2': []})
2021-11-09 08:20:46,169 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:46,171 | 3734 | tracing_of_pipeline.analyzer, analyze, 248 | WARNING | edge l1 is not terminal edge. The analysis done before may not have been done properly.
2021-11-09 08:20:46,398 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:48,166 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's2-eth2': [], 's1-eth2': []})
2021-11-09 08:20:48,168 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:48,169 | 3734 | tracing_of_pipeline.analyzer, analyze, 248 | WARNING | edge l1 is not terminal edge. The analysis done before may not have been done properly.
2021-11-09 08:20:48,174 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:50,166 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's2-eth2': [], 's1-eth2': []})
2021-11-09 08:20:50,168 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:50,169 | 3734 | tracing_of_pipeline.analyzer, analyze, 248 | WARNING | edge l1 is not terminal edge. The analysis done before may not have been done properly.
2021-11-09 08:20:50,173 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:52,160 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's2-eth2': [], 's1-eth2': []})
2021-11-09 08:20:52,161 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:52,162 | 3734 | tracing_of_pipeline.analyzer, analyze, 248 | WARNING | edge l1 is not terminal edge. The analysis done before may not have been done properly.
2021-11-09 08:20:52,164 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:54,165 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's2-eth2': [], 's1-eth2': []})
2021-11-09 08:20:54,166 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:54,166 | 3734 | tracing_of_pipeline.analyzer, analyze, 248 | WARNING | edge l1 is not terminal edge. The analysis done before may not have been done properly.
2021-11-09 08:20:54,169 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:56,159 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's2-eth2': [], 's1-eth2': []})
2021-11-09 08:20:56,159 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:56,160 | 3734 | tracing_of_pipeline.analyzer, analyze, 248 | WARNING | edge l1 is not terminal edge. The analysis done before may not have been done properly.
2021-11-09 08:20:56,164 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:20:58,158 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's2-eth2': [], 's1-eth2': []})
2021-11-09 08:20:58,159 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:20:58,160 | 3734 | tracing_of_pipeline.analyzer, analyze, 248 | WARNING | edge l1 is not terminal edge. The analysis done before may not have been done properly.
2021-11-09 08:20:58,163 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:21:00,158 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={'s1-eth1': [], 's2-eth2': [], 's1-eth2': []})
2021-11-09 08:21:00,159 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:21:00,160 | 3734 | tracing_of_pipeline.analyzer, analyze, 248 | WARNING | edge l1 is not terminal edge. The analysis done before may not have been done properly.
2021-11-09 08:21:00,164 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:21:00,394 | 3734 | tracing_net.net, stop_tracing, 541 | INFO | tracing stop
2021-11-09 08:21:00,395 | 3734 | tracing_of_pipeline.analyzer, stop_analyzing, 151 | INFO | stop analyzing interval. (last count = 31)
2021-11-09 08:21:00,411 | 3734 | tracing_net.packet_repository, pop, 73 | ERROR | Failed to pop from packet repository (repo={})
2021-11-09 08:21:00,411 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l2
2021-11-09 08:21:00,412 | 3734 | tracing_of_pipeline.analyzer, analyze, 248 | WARNING | edge l1 is not terminal edge. The analysis done before may not have been done properly.
2021-11-09 08:21:00,415 | 3734 | tracing_of_pipeline.analyzer, analyze, 251 | WARNING | No packet in edge l4
2021-11-09 08:21:00,415 | 3734 | tracing_of_pipeline.analyzer, analyze_rest, 163 | INFO | These packets could not be analyzed. {'l2': [], 'l1': [<Msg captured_interface=s1-eth2, sniff_timestamp=1636446019.5545619, pkt=<ARP_RAW Packet>, in_port=s1-eth2, in_phy_port=s1-eth2>, <Msg captured_interface=s1-eth2, sniff_timestamp=1636446019.606372, pkt=<ICMP_RAW Packet>, in_port=s1-eth2, in_phy_port=s1-eth2>, <Msg captured_interface=s1-eth2, sniff_timestamp=1636446019.6176686, pkt=<ICMP_RAW Packet>, in_port=s1-eth2, in_phy_port=s1-eth2>], 'l3': [<Msg captured_interface=s2-eth2, sniff_timestamp=1636446024.8289483, pkt=<ARP_RAW Packet>, in_port=2, in_phy_port=s1-eth2>], 'l4': []}
2021-11-09 08:21:00,416 | 3734 | tracing_of_pipeline.analyzer, analyze_rest, 164 | INFO | These OpenFlowMessage could not be analyzed. {'s1': [<src.ofproto.msg.MsgForOFMsg object at 0x7f75dc085250>, <src.ofproto.msg.MsgForOFMsg object at 0x7f75dc085310>, <src.ofproto.msg.MsgForOFMsg object at 0x7f75dc0854c0>], 's2': [<src.ofproto.msg.MsgForOFMsg object at 0x7f75dc085550>, <src.ofproto.msg.MsgForOFMsg object at 0x7f75dc0855b0>, <src.ofproto.msg.MsgForOFMsg object at 0x7f75dc0856d0>]}
